src/services/ui/src/pages/actions/RespondToRai.tsx

```diff
@@ -0,0 +1,372 @@
+                  </I.FormLabel>
+                  <I.Upload
+                    files={field?.value ?? []}
+                    setFiles={field.onChange}
+                  />
+                  <I.FormMessage />
+                </I.FormItem>
+              )}
+            />
+          ))}
+          {/*-------------------------------------------------------- */}
+          <I.FormField
+            control={form.control}
+            name="additionalInformation"
+            render={({ field }) => (
+              <I.FormItem>
+                <h3 className="font-bold text-2xl font-sans">
+                  Additional Information
+                </h3>
+                <I.FormLabel className="font-normal">
+                  Add anything else that you would like to share with CMS.
+                </I.FormLabel>
+                <I.Textarea {...field} className="h-[200px] resize-none" />
+                <I.FormDescription>4,000 characters allowed</I.FormDescription>
+              </I.FormItem>
+            )}
+          />
+          {/*-------------------------------------------------------- */}
+          <div className="my-2">
+            <i>
+              Once you submit this form, a confirmation email is sent to you and
+              to the original submitter.
+            </i>
+          </div>
+          {Object.keys(form.formState.errors).length !== 0 ? (
+            <Alert className="mb-6" variant="destructive">
+              Missing or malformed information. Please see errors above.
+            </Alert>
+          ) : null}
+          {form.formState.isSubmitting ? (
+            <div className="p-4">
+              <LoadingSpinner />
+            </div>
+          ) : null}
+          <div className="flex gap-2">
+            <I.Button
+              disabled={form.formState.isSubmitting}
+              type="submit"
+              className="px-12"
+            >
+              Submit
+            </I.Button>
+            <I.Button
+              type="button"
+              variant="outline"
+              onClick={() => setCancelModalIsOpen(true)}
+              className="px-12"
+            >
+              Cancel
+            </I.Button>
+            <Modal
+              open={successModalIsOpen}
+              onAccept={() => {
+                setSuccessModalIsOpen(false);
+                navigate(`/details?id=${id}`);
+              }}
+              onCancel={() => setSuccessModalIsOpen(false)}
+              title="Submission Successful"
+              body={
+                <p>
+                  Please be aware that it may take up to a minute for your
+                  submission to show in the Dashboard.
+                </p>
+              }
+              cancelButtonVisible={false}
+              acceptButtonText="Exit to Package Details"
+            />
+            <Modal
+              open={errorModalIsOpen}
+              onAccept={() => {
```

Comment:
We don't have a consistent Error handling pattern but we need one. I don't think this is the way, honestly. The copy will inevitably send droves of users to the helpdesk when we could probably just communicate there was a network issue or that our servers are at fault.